sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -970,6 +970,12 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val SHUFFLE_MANAGER_ENABLED = conf("spark.rapids.shuffle.enabled")
+    .doc("Enable the RAPIDS Shuffle Manager at runtime. When set to false, " +
+      "the built-in Spark shuffle will be used.")
```

Comment:
do we need to add comments about external shuffle service needing disabled?

Reply:
yes will add to this PR if that's OK. Seems relevant.

Reply:
Added more info with: https://github.com/NVIDIA/spark-rapids/pull/3107/commits/75c4dcf4eec64d272b05c1dc4709043fcc830213. Please let me know if that addresses the concern.

Reply:
looks good, thanks